exercises/conversions/try_from_into.rs

```diff
@@ -76,6 +73,11 @@ mod tests {
         let _ = Color::try_from((-1, -10, -256)).unwrap();
     }
     #[test]
+    #[should_panic]
+    fn test_tuple_sum() {
+        let _ = Color::try_from((-1, 255, 255)).unwrap();
```

Comment:
Disclamer: I am not maintainer of Rustlings just a modest suggestion:

Would have done something 
`assert!(Color::try_from((-1, 255, 255)).is_err())`
[playground link](like https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=use%20std%3A%3Aconvert%3A%3ATryFrom%3B%0A%0Astruct%20Color%3B%0Aimpl%20TryFrom%3C(i16%2C%20i16%2C%20i16)%3E%20for%20Color%20%7B%0Atype%20Error%20%3D%20String%3B%0Afn%20try_from(tuple%3A%20(i16%2C%20i16%2C%20i16))%20-%3E%20Result%3CSelf%2C%20Self%3A%3AError%3E%20%7B%0A%20%20%20%20Err(String%3A%3Afrom(%22No%22))%0A%7D%0A%0A%7D%0Afn%20main()%20%7B%0A%20%20%20%20assert!(Color%3A%3Atry_from((-1%2C%20255%2C%20255)).is_err())%0A%7D)

 I try to avoid unwrap in test because I hate the stack-trace it generate would prefers a `expect` or if the error is precise even an `assert_eq!` or a `assert!(expr.is_ok)` or `assert!(expr.is_err())` if it's hard to match value.

Reply:
Then we would need to rewrite every test with `is_err()` because every test uses `unwrap()`

Reply:
Hi y'all, this is a great addition. The desire to move from `.unwrap()` has been brought up in #549. If you're up for it, please feel free to rewrite the tests by substituting `.unwrap()` with `.is_err()` and removing the `#should_panic` directives.